src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,63 @@
+.stopwatch {
+  // background-color: cornflowerblue;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 80vmin;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__seconds-hand {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    position: absolute;
+    bottom: 50%;
+    z-index: 2;
+    transform-origin: bottom center;
+    animation-name: move;
+    animation-duration: 60s;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+  }
+
+  &__minutes-hand {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    bottom: 50%;
+    z-index: 1;
+    transform-origin: bottom center;
+    animation-name: move;
+    animation-duration: 3600s;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+  }
+
+  &__center {
+    height: 5vmin;
+    width: 5vmin;
+    border-radius: 5vmin;
+    background-color: #f6a603;
+    position: absolute;
+    z-index: 3;
+  }
+
+  @each $hand, $i in $hands {
+    &__#{$hand}-hand {
+      animation-duration: #{$i};
+    }
+  }
```

Comment:
The `@each` loop references `$hands` and `$i`, but these variables are not defined in the provided code. Ensure that `$hands` is defined as a map or list elsewhere in your SCSS files, or define it here if it's missing.